static/img/spectrum-config.svg

```diff
@@ -0,0 +1,18 @@
+<svg id="icon" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Not sure if it's worth splitting these out into separate vectors until we have a use case.

Reply:
They are inlined now and not split out into separate SVG files.